charts/opentelemetry-operator/Chart.yaml

```diff
@@ -12,4 +12,4 @@ maintainers:
   - name: jaronoff97
   - name: TylerHelmuth
 icon: https://raw.githubusercontent.com/cncf/artwork/a718fa97fffec1b9fd14147682e9e3ac0c8817cb/projects/opentelemetry/icon/color/opentelemetry-icon-color.png
-appVersion: 0.113.0
+appVersion: 0.114.0
```

Comment:
@araiu we also just pushed a patch for 0.114.0 to version 0.114.1 which we should use for the _operator_ app version here

Reply:
That was the initial version used, but followed @TylerHelmuth 's suggestion (https://github.com/open-telemetry/opentelemetry-helm-charts/pull/1448#discussion_r1871778845).
I agree to push to `0.114.1` to also get the fix from https://github.com/open-telemetry/opentelemetry-operator/issues/3463

Reply:
Ya that was the [collector version](https://github.com/open-telemetry/opentelemetry-helm-charts/pull/1448#discussion_r1871778845). There isn't a 0.114.1 of the collector.

Reply:
Ah, got mixed with the versions.
Pushed a new comit with the correct `0.114.1` for the operator and `0.114.0` for the collector